presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -120,10 +130,13 @@ public QueryState getState()
     }
 
     @JsonProperty
-    public boolean isScheduled()
-    {
-        return scheduled;
-    }
+    public ErrorType getErrorType() { return errorType; }
```

Comment:
Makes sense. It seemed odd when I saw that the method body is in the same line. Also I though that the auto formatting using Airlift will change it to the right format.